### PR TITLE
MIXEDARCH-356: bring intel control plane 4.14 job to 4.15 periodics

### PR DESCRIPTION
**Pull request by pkenchap**
[MIXEDARCH-377](https://issues.redhat.com//browse/MIXEDARCH-377) - Bring MAC Intel Control plane CI for 4.14 to 4.15 periodics 

updated the cron job to this time -20 7,19 * * * 
Modified OCP_VERSION from 4.14 to 4.15.

This PR for the issue - https://github.ibm.com/redstack-power/octopus/issues/455

### CODE REVIEW

**Review by prb112 — CHANGES_REQUESTED** [review]
Comments inline

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

File: ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json [hunk 1]
```diff
@@ -4,14 +4,16 @@
...
 			"Shilpa-Gokul",
 			"prb112",
+			"pkenchap",
 			"werled",
 			"multiarch-reviewers"
 		]
 	}
-}
\ No newline at end of file
+}
```
Comment (prb112) [inline ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json:19]: The metadata check is complaining about an extra return at the end. `diff -Naupr .//ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json /tmp/tmp.mFAw1KgdTt/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json`

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44651/pull-ci-openshift-release-master-step-registry-metadata/1715344464936964096

ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json

File: ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/OWNERS [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 - werled
 - multiarch-approvers
 reviewers:
 - Shilpa-Gokul
 - prb112
+- pkenchap
 - werled
-- multiarch-reviewers
\ No newline at end of file
+- multiarch-reviewers
```
Comment (prb112) [inline ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/OWNERS:12]: May need to check there is no new-line at the end of this file. Multiarch-reviewers shouldn't be flagged as a change

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml [hunk 3]
```diff
@@ -317,6 +317,40 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-serial
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
```
Comment (prb112) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:320]: This job https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44651/pull-ci-openshift-release-master-generated-config/1715344463456374784#1:build-log.txt%3A13
It's telling us that we're missing a change in ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml	and we'll have to update that too. We may need to update the cron specification in this file too... possibly.

You can see the additional configurations

```
- agent: kubernetes
+  cron: 20 7,19 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 6h0m0s
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: multiarch
+  labels:
+    ci-operator.openshift.io/cloud: ibmcloud-multi-ppc64le
+    ci-operator.openshift.io/cloud-cluster-profile: ibmcloud-multi-ppc64le
+    ci-operator.openshift.io/variant: nightly-4.15
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      - --report-credentials-file=/etc/report/credentials
+      - --secret-dir=/secrets/ci-pull-credentials
+      - --secret-dir=/usr/local/ocp-e2e-ibmcloud-ovn-multi-ppc64le-cluster-profile
+      - --target=ocp-e2e-ibmcloud-ovn-multi-ppc64le
+      - --variant=nightly-4.15
+      command:
+      - ci-operator
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /etc/boskos
+        name: boskos
+        readOnly: true
+      - mountPath: /secrets/ci-pull-credentials
+        name: ci-pull-credentials
+        readOnly: true
+      - mountPath: /usr/local/ocp-e2e-ibmcloud-ovn-multi-ppc64le-cluster-profile
+        name: cluster-profile
+      - mountPath: /secrets/gcs
+        name: gcs-credentials
+        readOnly: true
+      - mountPath: /secrets/manifest-tool
+        name: manifest-tool-local-pusher
+        readOnly: true
+      - mountPath: /etc/pull-secret
+        name: pull-secret
+        readOnly: true
+      - mountPath: /etc/report
+        name: result-aggregator
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: boskos
+      secret:
+        items:
+        - key: credentials
+          path: credentials
+        secretName: boskos-credentials
+    - name: ci-pull-credentials
+      secret:
+        secretName: ci-pull-credentials
+    - name: cluster-profile
+      secret:
+        secretName: cluster-secrets-ibmcloud-multi-ppc64le
+    - name: manifest-tool-local-pusher
+      secret:
+        secretName: manifest-tool-local-pusher
     - name: pull-secret
       secret:
         secretName: registry-pull-credentials 
```

Comment (pkenchap) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:320]: Added these in the `ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml ` .

File: ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/OWNERS (hunk 2 above)
Comment (pkenchap) [reply ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/OWNERS]: yes , it was added from the vim , I removed using the truncate command .

File: ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json (hunk 1 above)
Comment (pkenchap) [reply ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.metadata.json]: Removed the new line added from the truncate command .

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

**Comment by prb112** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

**Comment by prb112** [comment]
> `{"component":"checkconfig","error":"error loading prow config: error unmarshaling ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml: error converting YAML to JSON: yaml: line 14575: mapping values are not allowed in this context","file":"k8s.io/test-infra/prow/cmd/checkconfig/main.go:248","func":"main.main","level":"fatal","msg":"Validation failed","severity":"fatal","time":"2023-10-25T12:31:18Z"}`

*bad* 3 space

```
- agent: kubernetes
   cluster: build01
   cron: 20 7,19 * * *
```

*good* 2 space

```
- agent: kubernetes
  cluster: build04
  cron: 0 11 * * 0
  decorate: true
```

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

**Comment by prb112** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

**Comment by prb112** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

**Comment by pkenchap** [comment]
Closing the PR as new PR is opened for the same and will track the same in this PR - https://github.com/openshift/release/pull/45087

**Pull request closed without merging**